components/BottomNavigation/src/MDCBottomNavigationBar.m

```diff
@@ -496,6 +500,30 @@ - (void)setSelectedItem:(UITabBarItem *)selectedItem animated:(BOOL)animated {
   }
 }
 
+- (void)setItemsTitleVisibleTopPadding:(CGFloat)itemsTitleVisibleTopPadding {
+  if (_itemsTitleVisibleTopPadding == itemsTitleVisibleTopPadding) {
```

Comment:
For CGFloat, please use `MDCCGFloatEqual(a,b)` instead of directly equality comparisons.  Same in `-setItemsContentVerticalMargin:`